### PR TITLE
Fix sprintf call in AdminThemesController

### PR DESCRIPTION
**Pull request by prestamodule**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | As now, the %s is replaced with "htmlspecialchars"
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1896
| How to test?  | Check the Pull Request and select a theme.

### CODE REVIEW

File: controllers/admin/AdminThemesController.php [hunk 1]
```diff
@@ -428,8 +428,8 @@ public function renderOptions()
 
         $other_themes = $this->theme_repository->getListExcluding([$this->context->shop->theme->getName()]);
         if (!empty($other_themes)) {
-            $this->fields_options['theme'] = array(
-                'title' => sprintf($this->trans('Select a theme for the "%s" shop', array(), 'Admin.Design.Feature'), $this->context->shop->name),
+            $this->fields_options['theme'] = array(              
+                'title' => sprintf($this->trans('Select a theme for the "%s" shop', array('%s' => $this->context->shop->name), 'Admin.Design.Feature')),
```
Comment (antoin-m) [inline controllers/admin/AdminThemesController.php]: You may want to remove the call to `sprintf` if there is no need for it.

**Review by antoin-m — CHANGES_REQUESTED** [review]

**Comment by maximebiloe** [comment]
Hello @prestamodule,

The PR https://github.com/PrestaShop/PrestaShop/pull/7087 should solve issue more globally.
Could you test it and let us know?

Regards

**Comment by PrestaEdit** [comment]
Hi @maximebiloe 

It's seems not working with the PR you mention...

**Comment by maximebiloe** [comment]
Hello @prestamodule,

The PR https://github.com/PrestaShop/PrestaShop/pull/7194 solve the "htmlspecialchars" issue, so this one is no longer needed.
Thank you for your help.

Regards

**Pull request closed without merging**